.docs/Add-VSTeamTaskGroup.md

```diff
@@ -0,0 +1,83 @@
+<!-- #include "./common/header.md" -->
+
+# Add-VSTeamTaskGroup
+
+## SYNOPSIS
+
+<!-- #include "./synopsis/Add-VSTeamTaskGroup.md" -->
+
+## SYNTAX
+
+## DESCRIPTION
+
+<!-- #include "./synopsis/Add-VSTeamTaskGroup.md" -->
+
+## EXAMPLES
+
+### -------------------------- EXAMPLE 1 --------------------------
```

Comment:
Can you add a description to this example? I am not sure why if we are adding a group why you are calling Get-VSTeamTaskGroup first. I am guessing you are copying and existing Task Group? You can look at Add-VSTeamGitRepository for example with a description.